force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,18 +262,25 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
+        String result = '';
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
+        
+        for (Integer i = 1; i <= repeatCount; i++) {
+           if (i == repeatCount) {
+            result += inputString ;
+           }
+           else{
+            result += inputString + ';' + ' ';
+           }
```

Comment:
Try to use a for() loop  and loop all the way to imputString

for( Integer i = 0; .........